src/main/java/io/dropwizard/revolver/resource/RevolverRequestResource.java

```diff
@@ -255,6 +258,21 @@ private Response processRequest(String service, RevolverHttpApiConfig.RequestMet
                         msgPackObjectMapper)).build();
     }
 
```

Comment:
This is needed in userService to handle vulnerability
The userService needs to know the request is coming from api to performa validations
For internal calls, they skip the validation